File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    result_1 = str(a)+str(b)
    result_2 = str(b)+str(a)
    result_1 = int(result_1)
    result_2 = int(result_2)
    if result_1 > result_2:
        answer = result_1
    elif result_1 < result_2:
        answer = result_2
    elif result_1 == result_2:
        answer = result_1
    return answer

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String sum_1 = String.valueOf(a) + String.valueOf(b);
        String sum_2 = String.valueOf(b) + String.valueOf(a);
        

        if (Integer.valueOf(sum_1) > Integer.valueOf(sum_2))
            return Integer.valueOf(sum_1);
        else if (Integer.valueOf(sum_1)<Integer.valueOf(sum_2))
            return Integer.valueOf(sum_2);
        else
            return Integer.valueOf(sum_1);
    }
}